.github/workflows/labels.yaml

```
@@ -0,0 +1,24 @@
+name: Auto Label
+
+on:
+  pull_request:
+    types: [ opened, edited, reopened, ready_for_review, synchronize ]
```

```suggestion
    types: [ opened, synchronize ]
```

Maybe I misunderstand actions, but I think `ready_for_review` and `edited` do not imply a change in files, so this shouldn't need to be run? And if it runs on `opened` and `synchronize`, then I think `reopened` is taken care of?